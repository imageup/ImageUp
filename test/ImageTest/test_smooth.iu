function main() -> int
{
        image: a = read("./images/face1.jpg");
        
        image: b;
        prints("No change to pixel (0,1)");
        tuple: x = get_pixel(a, (0,1));
        prints("Before: " + StrParsef(x[0]) + " " + StrParsef(x[1]) + " " + StrParsef(x[2]));
        b = smooth(a, 0.95);
        tuple: y = get_pixel(a, (0,1));
        prints("After: " + StrParsef(y[0]) + " " + StrParsef(y[1]) + " " + StrParsef(y[2]));
		save("test.out.2.jpg", b);

        return 0;
}
