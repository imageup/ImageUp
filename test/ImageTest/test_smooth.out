No change to pixel (0,1)
Before: 233.000 239.000 238.000
After: 233.000 239.000 238.000
